server/routes/caseAdmin/curfewAddress/selectAddress.ts

```diff
@@ -50,7 +50,7 @@ export default class SelectAddressRoutes {
         findAddressUrl: paths.prison.assessment.enterCurfewAddressOrCasArea.findAddress({
           prisonNumber: req.params.prisonNumber,
         }),
-        formattedPostcode: this.formatPostcode(postcode),
+        formattedPostcode: this.formatPostcode(searchQuery),
```

Comment:
NBD but might make sense to update this name as it could be postcode or query

Reply:
I will so this now separately